js-wixcode-sdk/$w/UploadButton.service.json

```diff
@@ -83,32 +84,33 @@
         "extra":
           {  } },
       { "name": "fileType",
-        "labels": [],
+        "labels":
+          [ "changed" ],
         "get": true,
         "set": true,
         "type": "string",
         "locations":
-          [ { "lineno": 137,
+          [ { "lineno": 140,
               "filename": "UploadButton.es6" },
-            { "lineno": 168,
+            { "lineno": 171,
               "filename": "UploadButton.es6" } ],
         "docs":
-          { "summary": "Sets or gets the type of file that a user can upload.",
+          { "summary": "Sets or gets the type of file that a site visitor can upload.",
             "description":
               [ "Setting the `fileType` property sets the type of file that",
-                " a user can upload. The value of `fileType` can be set to either",
-                " `\"Image\"`, `\"Document\"`, or `\"Video\"`.",
+                " a site visitor can upload. The value of `fileType` can be set to ",
+                " `\"Image\"`, `\"Document\"`, `\"Video\"`, or `\"Audio\"`.",
                 "",
                 " Getting the `fileType` property returns the current type of",
-                " file that a user can upload.",
+                " file that a site visitor can upload.",
                 "",
-                " The file types of the images and documents that can be uploaded are",
+                " The file types that can be uploaded are",
                 " listed [here](https://support.wix.com/en/article/supported-media-file-types-and-file-sizes)." ],
             "links": [],
             "examples":
               [ { "title": "Get an upload button's type",
                   "body":
-                    [ "let fileType = $w(\"#myUploadButton\").fileType;  // \"Image\"" ],
+                    [ "let fileType = $w(\"#myUploadButton\").fileType;" ],
```

Comment:
Why did you remove the example value?

Reply:
Because it looks like only Image is OK.  I liked it better when it listed the 4 types. But you didn't like that.